### DISCUSSION TITLE
Update go2rtc to version 1.9.4

### DISCUSSION THREAD

**Discussion by devdems** [blakeblackshear/frigate#14000] [Ask A Question] [answered]
When can we get the current version of go2rtc which is 1.9.4. I'm using the latest image ghcr.io/blakeblackshear/frigate:0.14.1-tensorrt and the version of go2rtc is 1.9.2.

**Comment by NickM-27** [comment, marked as answer]
Go2rtc 1.9.4 has known major bugs, we will not use this version

**Reply by devdems** [reply]
@NickM-27 Do you have any instructions on updating it in the docker manually as there are certain bug fixes in 1.9.3 that you do not have in your image?

**Reply by NickM-27** [reply]
it's in the docs https://docs.frigate.video/configuration/advanced#custom-go2rtc-version

**Reply by devdems** [reply]
Working. Tnx!
